### PR TITLE
Media BOM update

### PR DESCRIPTION
**Pull request by goodmind**
https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object
https://html.spec.whatwg.org/multipage/history.html#history-3
https://html.spec.whatwg.org/multipage/web-sockets.html#binarytype
https://html.spec.whatwg.org/multipage/web-sockets.html#dom-websocket-send

https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API
https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker#Parameters

### CODE REVIEW

**Comment by dsainati1** [comment]
This has an error diff; do you think you could break this up into a couple smaller, more specific PRs, to make it easier to review?

**Comment by goodmind** [comment]
What errors?

**Comment by goodmind** [comment]
MIDI Access https://github.com/facebook/flow/pull/7804

File: lib/bom.js [hunk 1]
```diff
@@ -67,57 +66,116 @@ type Gamepad = {
...
-    title?: string;
-    text?: string;
-    url?: string;
+  title?: string;
+  text?: string;
+  url?: string;
 }
 
-declare class NavigatorCommon {
+type PermissionName = "geolocation" | "notifications" | "push" | "midi" | "camera" | "microphone" | "speaker" | "device-info" | "background-sync" | "bluetooth" | "persistent-storage" | "ambient-light-sensor" | "accelerometer" | "gyroscope" | "magnetometer" | "clipboard";
```
Comment (mroch) [inline lib/bom.js]: "clipboard" should be "clipboard-read" | "clipboard-write" according to https://w3c.github.io/clipboard-apis/#clipboard-permissions and https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API. and its own descriptor, I suppose

Comment (goodmind) [reply lib/bom.js]: Maybe I'm missing something, but in https://w3c.github.io/permissions/#enumdef-permissionname there isn't "clipboard-read" and "clipboard-write"

Comment (goodmind) [reply lib/bom.js]: Yeah, I tried using this API, and there isn't "clipboard"

**Pull request closed without merging**